build/config/tsa.json

```diff
@@ -0,0 +1,6 @@
+{
+    "instanceUrl": "https://microsoft.visualstudio.com",
+    "projectName": "OS",
+    "areaPath": "OS\\Windows Client and Services\\ADEPT\\E4D-Engineered for Developers\\SHINE\\Terminal",

```

Comment:
oh no they know what our area path is oh no

Reply:
oh no, they'll think we're engineering for developers oh no